src/Neo/Cryptography/Helper.cs

```diff
@@ -117,6 +118,30 @@ public static byte[] Sha256(this byte[] value)
             return sha256.ComputeHash(value);
         }
 
+        /// <summary>
+        /// Computes the 32-bit hash value for the specified byte array using the xxhash3 algorithm.
+        /// </summary>
+        /// <param name="value">The input to compute the hash code for.</param>
+        /// <param name="seed">The seed used by the xxhash3 algorithm.</param>
+        /// <returns>The computed hash code.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static uint XxHash3_32(this ReadOnlySpan<byte> value, uint seed = 0)
+        {
+            return (uint)XxHash3.HashToUInt64(value, seed);
```

Comment:
Avoid the overflow with HashCode.Combine?

Reply:
> Avoid the overflow with HashCode.Combine?

Taking lower 32bit is enough, I think.

Reply:
Should be! Unless it 2GB or more